.changeset/metal-drinks-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@segment/analytics-next': minor
```

Comment:
Should this be a patch, since it's not a user-facing feature?

Reply:
I saw it more as a "non-breaking enhancement" than a bug fix - so went with it

Reply:
Would vote for patch since it isn't a user-facing change to any API/API behavior.